query/src/main/scala/filodb/query/ResultTypes.scala

```diff
@@ -65,3 +65,28 @@ final case class QueryResult(id: String,
   }
 }
 
+sealed trait StreamQueryResponse extends NodeResponse with java.io.Serializable {
+  def id: String
+  def isLast: Boolean = false
+}
+
+final case class StreamQueryResultHeader(id: String,
+                                         resultSchema: ResultSchema) extends StreamQueryResponse
+
+final case class StreamQueryResult(id: String,
+                                   result: RangeVector) extends StreamQueryResponse
+
+final case class StreamQueryResultFooter(id: String,
+                                         queryStats: QueryStats = QueryStats(),
+                                         mayBePartial: Boolean = false,
+                                         partialResultReason: Option[String] = None) extends StreamQueryResponse {
+  override def isLast: Boolean = true
+}
+
+final case class StreamQueryError(id: String,
+                                  queryStats: QueryStats,
+                                  t: Throwable) extends StreamQueryResponse with filodb.core.ErrorResponse {
+  override def isLast: Boolean = true
+  override def toString: String = s"StrQueryError id=$id ${t.getClass.getName} ${t.getMessage}\n" +
```

Comment:
Can we change ``StrQueryError`` to ``StreamQueryError`` same as class name?

Reply:
Done